pkg/store/bucket.go

```diff
@@ -221,6 +226,14 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Help: "Compressed size of postings stored into cache.",
 	})
 
+	m.indexOperationDuration = promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+		Name: "thanos_bucket_store_index_operation_duration_seconds",
```

Comment:
[just a question] Have you considered two different metrics, one for postings and one for series? Like `thanos_bucket_store_index_series_lookup_duration_seconds` and ``thanos_bucket_store_index_postings_lookup_duration_seconds`.

Reply:
That's what I had done at first. I don't have any strong feelings about it. If you think it would be more clear to have 2 different metrics for this, I'm happy to change it.

Reply:
I don't have strong opinions too. We have different metrics for series/postings right now so, maybe, splitting it slightly more consistent with existing metrics but, again, no hard feelings here 😉